### ISSUE TITLE
Work Queue Python 3.7 SWIG Bindings missing required argument to Task._determine_file_flags in Task.specify_buffer 

### ISSUE THREAD

**Issue by jeffkinnison** [cooperative-computing-lab/cctools#2548]
# Details

When calling Task.specify_buffer in Python 3.7 (likely other versions as well), the following error occurs:

```
File "/afs/crc.nd.edu/user/j/jkinniso/Private/gypsum_dl_druid/cctools/lib/python3.7/site-packages/work_queue.py", line 2297, in specify_buffer
    flags = Task._determine_file_flags(flags, cache)
TypeError: _determine_file_flags() missing 1 required positional argument: 'failure_only'
```

## Environment

Standard CRC bash or tcsh environment with the latest version of CCTools built with SWIG 3.0.7 and

```
configure --prefix $HOME/cctools --with-python-path /opt/crc/p/python/3.7.3/gcc
make && make install
```

## Steps to Reproduce

With a CCTools install with Python 3.7 Work Queue bindings built, run

```
export PYTHONPATH="/path/to/cctools/lib/python3.7/site-packages:$PYTHONPATH"
python3 -c 'from work_queue import Task; task = Task("echo foo"); task.specify_buffer("bar", "baz.txt")'
```

## Notes

This appears to be the same bug that @tphung3 and I encountered when testing a fresh SHADHO install.

**Comment by dthain** [comment]
I believe @tphung3 is looking into this today.

**Comment by jeffkinnison** [comment]
Ah, great! I found it again while trying to roll out a new application on Condor, so it shouldn't be SHADHO-specific. The current workaround is to directly edit work_queue.py, and that seems to be enough to run WQ as expected until a fix is in.

**Comment by btovar** [comment]
Fixed by #2549
